R/client-cloud.R

```diff
@@ -66,15 +66,39 @@ cloudClient <- function(service, authInfo) {
       listRequest(service, authInfo, path, query, "applications")
     },
 
-    getApplication = function(applicationId) {
-      path <- paste("/applications/", applicationId, sep = "")
-      application <- GET(service, authInfo, path)
+    getApplication = function(applicationId, dcfVersion) {
```

Comment:
Would be clearer if we renamed the function arg `applicationId` to just `id`, since it could be either that of the application or the output?